server/ServerApp.cpp

```diff
@@ -1638,6 +1638,42 @@ void ServerApp::AddObserverPlayerIntoGame(const PlayerConnectionPtr& player_conn
     // TODO: notify other players
 }
 
+void ServerApp::DropPlayersEmpire(int player_id) {
+    m_player_empire_ids.erase(player_id);
```

Comment:
prefer `{}` on single line for single line function

Reply:
Rename -> `DropPlayerEmpire` ?

Reply:
May be even `DropPlayerEmpireLink` as it drops link between player and empire not empire itself.

Reply:
sure

Reply:
Fixed.